### DISCUSSION TITLE
Search Function

### DISCUSSION THREAD

**Discussion by ILikeToasters** [toutjavascript/Fooocus-Log-Viewer#21] [Q&A] [answered]
I have started using this and just tried the search function and it only seems to find content for the current day since I have installed it. Wondering if I am the only one with the issue. I would assume it would search all past images in the other folders. Maybe I am missing something.

**Comment by toutjavascript** [comment, marked as answer]
Thanks
You are right. 
Search is now updated and fully functional on release 1.5.6

**Reply by ILikeToasters** [reply]
Thanks! Love this tool!

**Reply by ILikeToasters** [reply]
Tested and working great!
